v2/datastream-to-spanner/terraform/samples/README.md

```diff
@@ -0,0 +1,43 @@
+## Terraform samples for live migration
+
+This repository provides samples for common scenarios users might have while trying to run a live migration to Spanner.
+
+Pick a sample that is closest to your use-case, and use it as a starting point, tailoring it to your own specific needs.
+
+## Other Sample Repositories
+
+The following sample repositories provide additional examples -
+
+1. [Sample environment setups](https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/main/v2/spanner-common/terraform/samples)
+2. [Bulk migration](https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/main/v2/sourcedb-to-spanner/terraform/samples)
+
+## List of examples
+
+1. [MySQL source to Spanner end-to-end](all-infrastructure/README.md)
```

Comment:
nit: i would rename this mysql-end-to-end/mysql-production-migration instead

Reply:
Done!